packages/dialtone-vue2/recipes/conversation_view/message_input/message_input.vue

```diff
@@ -182,20 +185,21 @@
             :class="[
               {
                 'dt-message-input__send-button--disabled': isSendDisabled,
-                'd-btn--circle': showSend.icon,
+                'd-btn--circle': hasSendIcon,
               },
             ]"
             :aria-label="showSend.ariaLabel"
             :aria-disabled="isSendDisabled"
             @click="onSend"
           >
             <template
-              v-if="showSend.icon"
+              v-if="hasSendIcon"
               #icon
             >
-              <dt-icon
-                :name="showSend.icon"
-                size="300"
+              <!-- @slot Slot for send button icon -->
+              <slot
+                name="sendIcon"
+                :icon-size="'300'"
               />
```

Comment:
This is not exactly the same way as it worked before right? If nothing is passed into the "showSend" prop or "sendIcon" slot it should render the "send" icon as default. You can do this by rendering the sendIcon within the slot. If the user wants to override it, they use the sendIcon slot

Reply:
You are right, thanks for catching this. Will fix.

Reply:
I changed it to have default content for the send icon. Also, `showSendIcon` will be true when there is no text since text and icon are exclusive.